### ISSUE TITLE
Link component not traversed with keyboard on MacOS+FF

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon#5532]
## Link not focusable with keyboard on MacOS with Firefox
This issue does **not** occur on Chrome and does **not** occur on Windows

## Environment

> MacOS Catalina 10.15.2

> Firefox 73.0.1 (64 Bit)

## Detailed description

> Carbon/React-version: I am using the carbon react storybook and thus whatever version is used there as of today.

> I expect the Link element to be focusable using the keyboard

> The Link element is not focusable

> All interactive elements must be reachable using keyboard.

## Steps to reproduce the issue

1. Open Firefox on MacOS
2. Navigate to here https://react.carbondesignsystem.com/?path=/story/link--default
3. Select the Link (see red rectangle in the screen shot below) with the mouse
4. Press TAB using the key to shift the focus to the next element in the focus chain
5. Press SHIFT+TAB to go back the focus chain and hence focus the Link element

Expectation: Link is focused
Reality: Link is not focused

## Additional information
![image](https://user-images.githubusercontent.com/51151716/75884357-f4603200-5e24-11ea-937d-5c2aca3eafcb.png)

**Comment by emyarod** [comment]
I'm not able to replicate the issue. This is what I'm seeing on Firefox 73+, macOS 10.14.6:

![Kapture 2020-03-04 at 10 12 30](https://user-images.githubusercontent.com/8265238/75899172-c963e700-5e00-11ea-9e4a-1215ee3fac73.gif)

**Comment by Daniel-Schulz-Private** [comment]
Possibly introduced with MacOS 10.15.2?

This is what I see if I
1. click on "Link" with the mouse
2. TAB away from "Link"
3. SHIFT TAB back to "Link"

after 3. the focus seems to be on the show-container of the storybook.

![out](https://user-images.githubusercontent.com/51151716/75963213-60927280-5ec5-11ea-818e-762f806ec2da.gif)

**Comment by Daniel-Schulz-Private** [comment]
It seems this might be an issue with my installation. I can't TAB to any links in FF, not only with carbon/react-links. Basically on any website.

So I guess you may close this issue.

Thank you for your time.

**Comment by Daniel-Schulz-Private** [comment]
**FYI:** It seems on Firefox/MacOS I had to either 
- manually configure a particular value, like described here https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox

- or have TAB enabled in Preferences/Keyboard/Shortcuts like so
<img width="633" alt="image" src="https://user-images.githubusercontent.com/51151716/75972836-65aaee00-5ed4-11ea-8420-20b06926d72c.png">

I do not know why this was not enabled on my Mac.

**Comment by emyarod** [comment]
I just updated to 10.15.3 and focus changing was still working as expected, so it was probably due to that setting on your machine
